Simulation/TSoG_X1_Sim.m

```diff
@@ -0,0 +1,95 @@
+% Based off of https://github.com/TSoGiants/MECH2103-Computational-Aerodynamics
+%  TSoG_X1_Sim is the main function to run the X1 simulation. It can be run with
+## or without an input Test Case (the sim will run with default inputs if no Test
+## Case is provided)
+
+
+function [ Results ] = TSoG_X1_Sim( TestCase )
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  %                        Simulation Setup
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  if nargin == 0
+    TestCase.Name        = 'Default';
+    TestCase.Description = 'Default Test Case for X1 Simulation';
+    P0 = [0, 10]; % Initial position (m)
```

Comment:
Just a few clarity things here:
- You may want to add `[X Y]` here like you did with `[Pitch]`.
- There also used to be a comment here explaining what the X and Y axis were (left/right and up/down). We still want that.

And finally, we probably want both of these in the `TemplateTestCase.m` file too.

Reply:
Agree on all these, will do.

Reply:
Added changes detailed above, and added them to the TemplateTestCase as well